File: Algorithms/Warmup/time-conversion/time-conversion.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
time = raw_input()
hours = int(time[0:2])
if time[8:] == 'PM' and time[0:2] == '12':
    hours = 12
elif time[8:] == 'PM':
    hours += 12
print str(hours) + time[2:8]

File: Algorithms/Warmup/time-conversion/time-conversion.js
function processData(input) {
    //Enter your code here
    var time = input;
    var hours = parseInt(time.substr(0,2));
    if ((time.endsWith('PM')) && (hours == '12')){
        hours = 12;
    } else  if (time.endsWith('PM')) {
        hours += 12;
    }
    console.log(hours + time.substr(2,6));
} 

process.stdin.resume();
process.stdin.setEncoding("ascii");
_input = "";
process.stdin.on("data", function (input) {
    _input += input;
});

process.stdin.on("end", function () {
   processData(_input);
});

